zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/GUI.java

```diff
@@ -96,7 +96,17 @@ public Object password(String id) {
     return params.get(id);
   }
 
+  /**
+   * Create a select form.
+   *
+   * <p>If {@code defaultValue} is a {@link java.util.Collection} the first
+   * element will be used as the actual default value.</p>
+   */
   public Object select(String id, ParamOption[] options, Object defaultValue) {
+    if (defaultValue instanceof Collection && !(defaultValue instanceof String)) {
```

Comment:
```suggestion
    if (defaultValue instanceof Collection) {
```
Checking for the string class seems unnecessary to me.